### DISCUSSION TITLE
In which specification version were vulnerability references introduced?

### DISCUSSION THREAD

**Discussion by vbazurtob** [CycloneDX/specification#420] [Q&A] [answered]
Hi,

I would like to know from which version of the CycloneDX specification were vulnerability references introduced?
Here is an example:

https://github.com/CycloneDX/specification/blob/master/tools/src/test/resources/1.4/valid-vulnerability-1.4.xml#L18

**Comment by stevespringett** [comment, marked as answer]
That capability was introduced in CycloneDX v1.4.

**Comment by prabhu** [comment]
I think when a new attribute gets introduced to the spec, we can start adding `since <version>` to the help text.
